server/publicapi/items/service.py

```diff
@@ -146,7 +148,7 @@ def _process_fetched_object(self, document):
             document.pop(field_name, None)
 
         if 'renditions' in document:
-            for k, v in document['renditions'].items():
+            for _k, v in document['renditions'].items():
```

Comment:
Should probably be named simply `_`, denoting an unused variable (in contrast to a variable name prefixed with an underscore, denoting a "private" variable).